spring-petclinic/scripts/petclinic-deploy-openshift.sh

```diff
@@ -87,6 +89,46 @@ if [ -z "${NAMESPACE}" ]; then
 	NAMESPACE="${DEFAULT_NAMESPACE}"
 fi
 
+# Check if the memory request/limit has unit. If not ask user to append the unit
+case "${MEM_REQ}" in
```

Comment:
Suggest you create a function for checking the memory unit and call it for both `MEM_REQ` and `CPU_REQ`.